bandwidth-tps/Cargo.toml

```diff
@@ -13,11 +13,15 @@ log = "0.4.6"
 pubsub-client = { path = "../pubsub-client", version = "0.2.0" }
 rand = "0.6.5"
 rand_chacha = "0.1.1"
-rayon = "1.0.3"
-solana-client = "0.18.0"
-solana-drone = "0.18.0"
-solana-logger = "0.18.0"
-solana-sdk = "0.18.0"
+rayon = "1.2.0"
+solana-bpf-loader-api = { git = "https://github.com/solana-labs/solana", version = "0.19.0-pre0" }
```

Comment:
Should wait to merge this until solana v0.19.0 is released, so we can use crates.io?

Reply:
Sure, fine with me!